internal/commands/push/command_test.go

```diff
@@ -1029,11 +1029,135 @@ Try instead: realm-cli push --local testdata/dependencies --remote appID --inclu
 	})
 }
 
+func TestPushHandlerCreateNewApp(t *testing.T) {
```

Comment:
[mega-nit] we are testing the version/appStructure functionality in `internal/local/app_test.go` with `"mismatched version in the config file should fail to find app"` so it might make sense to rename this to `TestPushHandlerCreateNewAppNested` but as @makesitgo likes to say, "I won't die on this hill"

Reply:
Before this PR, even if an app had a `config.json` with the correct version number, `push` would write to a `realm_config.json`.
So this test is still checking that if the user has a correctly configured `config.json` that the command writes to the same `config.json`.